### PR TITLE
Filament Profile `README.md`

### PR DESCRIPTION
**Pull request by Z3d0X**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
     <img src="https://user-images.githubusercontent.com/41773797/131910226-676cb28a-332d-4162-a6a8-136a93d5a70f.png" alt="Banner" style="width: 100%; max-width: 800px;" />
 </p>
 
-<!--
+## About Filament
 
-**Here are some ideas to get you started:**
+[Filament](https://filamentphp.com/) is an open source collection of tools for rapidly building beautiful TALL stack apps, designed for humans.
```
Comment (zepfietje) [inline profile/README.md:7]:
```suggestion
[Filament](https://filamentphp.com) is an open source collection of tools for rapidly building beautiful TALL stack apps, designed for humans.
```

Comment (zepfietje) [inline profile/README.md:7]:
```suggestion
[Filament](https://filamentphp.com/) is a collection of tools for rapidly building beautiful TALL stack apps, designed for humans.
```

File: profile/README.md [hunk 2]
```diff
@@ -2,13 +2,19 @@
...
 
-🙋‍♀️ A short introduction - what is your organization all about?
-🌈 Contribution guidelines - how can the community get involved?
-👩‍💻 Useful resources - where can the community find your docs? Is there anything else the community should know?
-🍿 Fun facts - what does your team eat for breakfast?
-🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
--->
+## Our Packages
```
Comment (zepfietje) [inline profile/README.md:9]:
```suggestion
## Packages
```

File: profile/README.md [hunk 3]
```diff
@@ -2,13 +2,19 @@
...
+
+- [Admin Panel](https://filamentphp.com/docs/admin) - A fully-featured Laravel admin panel.
+- [Form Builder](https://filamentphp.com/docs/forms) - An intuitive Laravel form builder.
+- [Table Builder](https://filamentphp.com/docs/tables) - An interactive Laravel table builder.
+- [Notifications](https://filamentphp.com/docs/notifications) - Powerful Laravel notifications.
+
+## Additional Resources
```
Comment (zepfietje) [inline profile/README.md:16]:
```suggestion
## Resources
```

**Review by zepfietje — CHANGES_REQUESTED** [review]
Thanks, @Z3d0X!

Just a few tweaks:

**Comment by Z3d0X** [comment]
Hi @zepfietje, I've tried to place important links with a short description.

I do think we need a few more words in "About Filament" section, but I'm not sure what to say, any thoughts? Other than this, do you think anything else is missing?

I think its best to this short & simple.

**Comment by zepfietje** [comment]
Looks good to me right now!

**Pull request merged**